scripts/ci/mutation_pilot.py

```
@@ -0,0 +1,303 @@
+        return json.loads(STATS_FILE.read_text(encoding="utf-8"))
+    finally:
+        SETUP_CFG.unlink(missing_ok=True)
+
+
+def _kill_stragglers() -> None:
+    """Reap mutmut children left behind by a deadlocked fork.
+
+    ``subprocess`` kills the process it started; the forked grandchildren are
+    not in that tree and would otherwise sit around holding the sandbox.
+    """
+    subprocess.run(["pkill", "-f", "mutmut run"], check=False, capture_output=True)
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)